readme.md

```diff
@@ -209,7 +240,7 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 * **Question:** Try commenting out `body-parser` in your application. What happens and why?
 
 * **Your Answer:**
-
+Fails to parse in Postman.
```

Comment:
Fails to parse the body in the server, and subsequently fails to respond to Postman